### PR TITLE
Issue #9180: Update restConnector-2.0 description to mention reader a…

### LINKED ISSUE #9180: Update restConnector-2.0 feature description.
The restConnector-2.0 feature description needs to be updated to take into account the new reader-role.

There is also a type as the restConnector-2.0 feature doesn't supersede itself, but it does supersede restConnector-1.0.

Possibly might need to update the description on restConnector-1.0 as well.

### PR DESCRIPTION
**Pull request by jvanhill**
fixes #9180

### CODE REVIEW

**Comment by jvanhill** [comment]
@ManasiGandhi - I am not sure I see your changes. Can you elaborate?

File: dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 description=A secure, REST administrative connector that can be used locally or remotely using any Java \
 developer kit. This connector enables remote access from a Java client or directly through an HTTPS call. \
-It is protected by a single administrator role and requires SSL to keep the communication confidential. \
+It is protected by the administrator and the reader role and requires SSL to keep the communication confidential. \
```
Comment (ManasiGandhi) [inline dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties]: Change,
"It is protected by the administrator and the reader role and requires SSL to keep the communication confidential. "
to
"The administrator and reader roles protect the connector and it requires an SSL protocol to keep the communication confidential."

**Comment by ManasiGandhi** [comment]
@jvanhill Sorry for the confusion. Can you see the changes now?

File: dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties (hunk 1 above)
Comment (jvanhill) [reply dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties]: Ok. Those changes look good. I will update the text.

Comment (jvanhill) [reply dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties]: I am going to modify SSL to TLS and remove protocol as TLS implies a protocol and it sounds redundant.

Comment (ManasiGandhi) [reply dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties]: Okay. Thanks.

**Comment by andrea-pichler** [comment]
No message review requested from L2

File: dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties [hunk 2]
```diff
@@ -16,7 +16,7 @@
 
 description=A secure, REST administrative connector that can be used locally or remotely using any Java \
 developer kit. This connector enables remote access from a Java client or directly through an HTTPS call. \
-It is protected by a single administrator role and requires SSL to keep the communication confidential. \
+The administrator and reader roles protect the connector and it requires TLS to keep the communication confidential. \
```
Comment (ManasiGandhi) [inline dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties]: Small suggestion,
Change,
The administrator and reader roles protect the connector and it requires TLS to keep the communication confidential.
to,
The administrator and reader roles protect the connector. The connector requires TLS to keep the communication confidential.

File: dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties (hunk 1 above)
Comment (ManasiGandhi) [reply dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties]: @jvanhill Just to clarify. With reference to  'The administrator and reader roles protect the connector '; is it still s single administrator role and an additional or optional reader role?

Comment (jvanhill) [reply dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties]: @ManasiGandhi - These are two separate administrative roles. Both are optional. The administrator-role has all permission that the reader-role has.

File: dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties (hunk 2 above)
Comment (jvanhill) [reply dev/com.ibm.websphere.appserver.features/visibility/public/restConnector-2.0/resources/l10n/com.ibm.websphere.appserver.restConnector-2.0.properties]: @ManasiGandhi - Will make the change.

**Comment by jvanhill** [comment]
#build

**Comment by jvanhill** [comment]
#run-libby-bot

**Pull request merged**